scene/resources/particle_process_material.cpp

```diff
@@ -1567,16 +1567,25 @@ void ParticleProcessMaterial::set_emission_shape(EmissionShape p_shape) {
 	emission_shape = p_shape;
 	notify_property_list_changed();
 	_queue_shader_change();
+#ifdef TOOLS_ENABLED
+	emit_changed();
+#endif
```

Comment:
All affected properties should have this documented, because emitting the signal only in the editor is an exception. 
Whether it should at project run-time as well is debatable. If anything, all other particle properties should make note that `changed` is not emitted for them, but that's for another time.

Something like `[b]Note:[/b] Only in the editor, changing this property emits the [signal Resource.changed] signal`.

Reply:
Done as suggested 👍🏻 The signal is only emitted in the editor for performance reasons. This was discussed during rendering meeting with Clay and QbieShay